run.go

```diff
@@ -75,7 +75,11 @@ func Run(o Options) (err error) {
 
 	// Init window
 	var w *astilectron.Window
-	if w, err = a.NewWindow(filepath.Join(a.Paths().BaseDirectory(), resourcesPath, "app", o.Homepage), o.WindowOptions); err != nil {
+	var url = filepath.Join(a.Paths().BaseDirectory(), "resources", "app", o.Homepage)
```

Comment:
Could you replace `"resources"` by `resourcesPath`, changes have been made to that part by a previous PR

Reply:
I would rather see something like this:

```go
var url = o.Homepage
if !strings.HasPrefix(o.Homepage, "http://") && !strings.HasPrefix(o.Homepage, string(filepath.Separator)) {
  url = filepath.Join(a.Paths().BaseDirectory(), resourcesPath, "app", o.Homepage)
}
```

Reply:
I think shoule use ```http```, because more site use https protocol.

I can't find ```resourcesPath``` defined in anywhere, It seems previous PR not merged.